docs/source/conf.py

```diff
@@ -180,7 +180,7 @@
 html_logo = "_static/ScribeDataLogo.png"
 html_theme_options = {
     "logo_only": True,
-    "display_version": False,
+    "display_version": True,
```

Comment:
@andrewtavis here is the fix for #95. Apology for my mistake. :(

Reply:
All good, @mhmohona! Happens :)